src/main/include/commands/IntakeOn.h

```diff
@@ -7,10 +7,9 @@
 
 #pragma once
 
+#include <subsystems/Intake.h>
```

Comment:
Our subsystem includes use `"`

```suggestion
#include "subsystems/Intake.h"
```